modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/Run.tsx

```diff
@@ -92,7 +61,7 @@ const Runs = () => {
 						},
 					],
 				}}
-				transformData={transformData}
+				transformData = {getRunTransformData}
```

Comment:
Please, remove these spaces.
```suggestion
				transformData={getRunTransformData}
```